### PR TITLE
Refactor and corrected errors.

### PR DESCRIPTION
**Pull request by RandomGenericUsername**
- Modified Buffer size on UbiProtocolHandler to avoid errors. see https://github.com/ubidots/ubidots-esp32/commit/09cd6f6eaba825392d6e29807ebc54770e2d72e4.
- Refactored wifiConnect method to match the changes on the examples .

### CODE REVIEW

File: src/UbiProtocolHandler.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ void UbiProtocolHandler::buildHttpPayload(char *payload) {
   sprintf(payload, "{");
 
   for (uint8_t i = 0; i < _current_value;) {
-    char str_value[20];
+    char str_value[25];
```
Comment (dsepulvedar) [inline src/UbiProtocolHandler.cpp:146]: What error does this prevent?

File: examples/SendValuesTCP/SendValuesTCP.ino [hunk 2]
```diff
@@ -25,23 +27,24 @@ Ubidots ubidots(UBIDOTS_TOKEN, UBI_TCP);
...
  * Main Functions
  ****************************************/
-
 void setup() {
   Serial.begin(115200);
-  ubidots.wifiConnect(WIFI_SSID, WIFI_PASS);
-  // ubidots.setDebug(true);  // Uncomment this line for printing debug messages
+  Ubidots::wifiConnect(WIFI_SSID, WIFI_PASS);
+  ubidots = new Ubidots(UBIDOTS_TOKEN, UBI_TCP);
+  //ubidots->setDebug(true); //Uncomment this line for printing debug messages
```
Comment (dsepulvedar) [inline examples/SendValuesTCP/SendValuesTCP.ino]: For the sake of styling in all comment => "// <your comment>"

File: src/UbiProtocolHandler.cpp (hunk 1 above)
Comment (RandomGenericUsername) [reply src/UbiProtocolHandler.cpp:146]: The errors described in this PR for the esp32: https://github.com/ubidots/ubidots-esp32/commit/09cd6f6eaba825392d6e29807ebc54770e2d72e4

File: examples/SendValuesTCP/SendValuesTCP.ino (hunk 2 above)
Comment (RandomGenericUsername) [reply examples/SendValuesTCP/SendValuesTCP.ino]: Solved in the latest commit

**Review by crisap94 — APPROVED** [review]

**Pull request merged**